examples/tutorials/Part 12 bis - Encrypted Training on MNIST.ipynb

```diff
@@ -0,0 +1,664 @@
+{
```

Comment:
<p>Can you comment more about <code>requires_grad</code>, <code>fix_precision</code> and <code>share</code>? Maybe a paragraph highlighting why they are needed and a intuition on what they are doing / why needed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/2411/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>I will, should I also remove this part of the demo, like just call a utility file? I have the feeling that many people asked why sending the data to workers at the beginning (in the FL demos especially), so maybe keeping this magic and moving this to a .py file would be better, what do you think?</p>